pkg/app/AppListingService.go

```diff
@@ -806,6 +941,15 @@ func (impl AppListingServiceImpl) FetchOtherEnvironment(ctx context.Context, app
 		} else {
 			env.ChartRefId = chart.ChartRefId
 		}
+
+		gitCommits, exists := gitCommitsWithArtifacts[env.CiArtifactId]
+		if exists {
```

Comment:
rewrite it for only one assignment and one initialisation

Reply:
solved